lib/Finance/Quote/FinanceAPI.pm

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/perl -w
+# vi: set ts=2 sw=2 noai ic showmode showmatch:  
+#
+#    Copyright (C) 2024, Bruce Schuck <bschuck@asgard-systems.com>
+#
+#    This program is free software; you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation; either version 2 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program; if not, write to the Free Software
+#    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+#    02110-1301, USA
+
+#    Changes:
+#    Initial Version: 2024-08-31, Bruce Schuck
+
+package Finance::Quote::FinanceAPI;
+
+use strict;
+use warnings;
+
+use Encode qw(decode);
+use HTTP::Request::Common;
+use JSON qw( decode_json );
+
+use constant DEBUG => $ENV{DEBUG};
```

Comment:
Only recommendation here is to change it to:

`use constant DEBUG => $ENV{FQ_FINANCEAPI_DEBUG} || $ENV{FQ_DEBUG} || $ENV{DEBUG};`

Reason is that there are too many things out there that use DEBUG env var for all sort of things generically which obsefucates debug messages. This will allow to tune for specific emitter as much specific as needed to troubleshoot.